cloud/fabfile.py

```diff
@@ -49,11 +49,7 @@ def proxy():
 
 def _is_hg():
     """ Determines if the project is hg controlled """
-    try:
-        local("hg identify")
-        return True
-    except:
-        return False
+    return False
```

Comment:
what is the purpose of this change? It is functionally incorrect, when run in the FB repo this function should return True.